README.md

```diff
@@ -1 +1,97 @@
-# bitrise-step-decrypt-file
\ No newline at end of file
+# Decrypt file
+
+You can use this step to decrypt a file in your repository or stored on bitrise.io or other provider (with the File Downloader step).
```

Comment:
I would just say: `You can use this step to decrypt a file.`

Otherwise, we need to explain it in details (`You can store the file anywhere on the web but you have to download it during your build, using File Downloader step for example`)